lib/utils/dom.js

```diff
@@ -63,12 +63,6 @@ click.isProgrammaticEvent = e => {
 	return e.clientX === 1 && e.clientY === 1;
 };
 
-export function mousedown(obj, button = 0) {
```

Comment:
😜 

![image](https://cloud.githubusercontent.com/assets/7673145/18492344/e2bd2316-79d8-11e6-903a-f4c0489113c3.png)

s/a use/any future use/ to clarify